devices.php

```diff
@@ -2058,11 +2058,11 @@ function setPreferredLayout() {<?php if(isset($_COOKIE["layout"]) && strtolower(
 	<div class="table">
 		<div>
 		  <div><label for="APIUsername">'.__("API Username").'</label></div>
-		  <div><input autocomplete="off" type="text" name="APIUsername" id="APIUsername" value="'.$dev->APIUsername.'"></div>
+		  <div><input autocomplete="new-password" type="text" name="APIUsername" id="APIUsername" value="'.$dev->APIUsername.'"></div>

```

Comment:
I don't think new-password is appropriate for a user name and is exploiting bug behavior to prevent entry of the username.  This is likely not wise.